config/locales/en.yml

```diff
@@ -1673,8 +1673,8 @@ en:
         your_note_html: "%{commenter} has reactivated one of your map notes near %{place}."
         commented_note: "%{commenter} has reactivated a map note you have commented on. The note is near %{place}."
         commented_note_html: "%{commenter} has reactivated a map note you have commented on. The note is near %{place}."
-      details: "More details about the note can be found at %{url}."
-      details_html: "More details about the note can be found at %{url}."
+      details: "Reply or learn more about the changeset at %{url}."
```

Comment:
This looks like a copy+paste error - you’ve changed note to change set here (twice)

Reply:
woops, fixed